src/types/shop.ts

```diff
@@ -24,3 +21,14 @@ export type CheckoutOptions = {
   fee?: number
   extraData?: string
 }
+
+export type ShopConfiguartion = {
+  address: Nimiq.Address
+  name: string
+  logo?: string
+  id: string
+  publicKey: string
+  fee?: number
+  live?: boolean
```

Comment:
[nickpick] `isLive` instead of `live`?

Reply:
Was `production` before. No strong feeling for one or the other. Would leave it as `live`, could be `production` also, just picked a shorter term, trying to use the easiest, most natural words. Wdyt?

Reply:
>> according to conversion, we just leave it as is.